test_runner/regress/test_pageserver_crash_consistency.py

```diff
@@ -93,15 +95,11 @@ def test_local_only_layers_after_crash(neon_env_builder: NeonEnvBuilder, pg_bin:
     # wait for us to catch up again
     wait_for_last_record_lsn(pageserver_http, tenant_id, timeline_id, lsn)
 
-    pageserver_http.timeline_compact(tenant_id, timeline_id)
-
-    # give time for log flush
-    time.sleep(1)
+    pageserver_http.patch_tenant_config_client_side(tenant_id, {"compaction_threshold": 3})
```

Comment:
No need to patch again b/c L47 already patched it?

Reply:
33b56f239